apps/src/applab/api.js

```diff
@@ -288,51 +288,57 @@ exports.clearTimeout = function (blockId, timeoutId) {
 };
 
 
-exports.readSharedValue = function(blockId, key, onSuccess, onError) {
+exports.readRemoteValue = function(blockId, key, visibility, onSuccess, onError) {
   return Applab.executeCmd(blockId,
-                           'readSharedValue',
+                           'readRemoteValue',
                            {'key':key,
+                            'visibility': visibility,
```

Comment:
The validity of the visibility parameter is enforced in the AppStorage layer. Currently, the valid values for this string happen to be the same as the prefixes used by the underlying storage API.